src/Migrator.php

```diff
@@ -5,7 +5,11 @@
 use Illuminate\Support\Arr;
 use Illuminate\Http\Request;
 use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Event;
```

Comment:
Don't need this line since you used the helper, but I think we should use the facade.

Reply:
We should 😉

Reply:
👍